code/modules/mob/language.dm

```diff
@@ -180,6 +180,20 @@
 		new_name += ..(gender,1)
 	return new_name
 
+/datum/language/vulpkanin
+	name = "Canilunzt"
+	desc = "The guttural language spoken and utilized by the inhabitants of Vazzend system, composed of growls, barks, yaps, and heavy utilization of ears and tail movements.Vulpkanin speak this language with ease."
+	speech_verb = "rawrs"
```

Comment:
which were set to that because "muh cute x3 *POMF*", and is still unprofessional